js/modules/k6/ws/ws.go

```diff
@@ -123,24 +123,19 @@ func (mi *WS) Exports() modules.Exports {
 // Connect establishes a WebSocket connection based on the parameters provided.
 // TODO: refactor to reduce the method complexity
 //nolint: funlen,gocognit,gocyclo,cyclop
-func (mi *WS) Connect(url string, args ...goja.Value) (*WSHTTPResponse, error) {
+func (mi *WS) Connect(url string, paramsV, callableV goja.Value) (*WSHTTPResponse, error) {
 	ctx := mi.vu.Context()
 	rt := mi.vu.Runtime()
 	state := mi.vu.State()
 	if state == nil {
 		return nil, ErrWSInInitContext
 	}
 
-	// The params argument is optional
-	var callableV, paramsV goja.Value
-	switch len(args) {
-	case 2:
-		paramsV = args[0]
-		callableV = args[1]
-	case 1:
-		paramsV = goja.Undefined()
-		callableV = args[0]
-	default:
+	if callableV == nil {
```

Comment:
```suggestion
    // The params argument is optional
	if callableV == nil {
```
I'm not sure the next time I will be able to understand **why** we are doing it. So, I would prefer to maintain the comment explaining it.

Reply:
the reason for this changes was fixed upstream so I am dropping them